recipes/hdlib/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "hdlib" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a29590b61bf5d192a7e969379c58fea1fc3eae96582bfdd479aba0aa50b79a8c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - numpy >=1.22.3
+
+test:
+  imports:
+    - hdlib
+
+about:
+  home: https://github.com/fabio-cumbo/hdlib
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
```suggestion
  license_file: LICENSE
```

Reply:
Latest conda-build can recognize the license in the recipe folder without that. If you saw that message in our CIs it is quite odd. If you saw that locally you need to upgrade your conda/conda-build.

Reply:
The error message appeared during CIs actually..

Reply:
It worked! Let me know if you think it could be merged now
Thanks @ocefpaf